README.md

```diff
@@ -180,6 +180,75 @@ tasks.create('executeRestApiSpikeTest', RunQalipsis) {
 }
 ```
 
+### Executing Campaigns
+The `qalipsisRunCampaign` creates and executes a new campaign in the cloud.
+
+### Kotlin
+
+```kotlin
+tasks {
+    create("qalipsisRunCampaign", CloudRunQalipsis::class.java) {
+        campaign {
+            campaignName.set("Dummy campaign configuration")
```

Comment:
Move the campaign name as argument of the function campaign, to have campaign("Dummy campaign configuration") {
And find a more friendly name than `Dummy campaign configuration`